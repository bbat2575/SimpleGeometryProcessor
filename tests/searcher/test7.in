0, 0
0.0, 0.0
-0, -0
+0, +0
-0.0, -0.0
+0.0, +0.0
000000.00, 000000.00
-000000.00, -0000000.00
+000000.00, +0000000.00
