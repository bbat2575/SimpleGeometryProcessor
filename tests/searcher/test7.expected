read 9 points
0.00, 0.00
0.00, 0.00
0.00, 0.00
This is not a triangle
